### DISCUSSION TITLE
为什么我的这边完全失效啊

### DISCUSSION THREAD

**Discussion by z3597666** [loonghao/photoshop-python-api#205] [Q&A]
py:3.11
就是设置背景颜色
2018-2022的ps版本全部不生效也没有报错，就卡在哪里  然后弹出ps窗口  然后看不到设置前背景颜色

**Comment by loonghao** [comment]
还未在3.11的环境测试过，你试试3.7是否可行

**Reply by loonghao** [reply]
@z3597666 如果可以得话请把相关得错误截图发出来看看
